Source/Mod/Editor/Map/FujiMapWriter.cs

```diff
@@ -0,0 +1,150 @@
+using System.Reflection;
+using System.Reflection.PortableExecutable;
+using System.Runtime.InteropServices;
+using System.Runtime.Serialization.Formatters.Binary;
+using MonoMod.Utils;
+
+namespace Celeste64.Mod.Editor;
+
+public static class FujiMapWriter
```

Comment:
Wonder if it might make sense to merge the FujiMap reader file and this FujiMapWriter file together, since they're working with a lot of the same data.